netbox/utilities/templatetags/form_helpers.py

```diff
@@ -46,13 +58,33 @@ def widget_type(field):
 #
 
 @register.inclusion_tag('form_helpers/render_field.html')
-def render_field(field, bulk_nullable=False, label=None):
+def render_field(field, bulk_nullable=False, label=None, table=None, request=None):
```

Comment:
This is too much to cram into the general-purpose `render_field` tag. If a custom template tag is necessary (and it might not be), we should create a separate one specifically for table headers.

Reply:
I will pull it out into a separate tag.  It originally was but I didn't want to duplicate too much.

I will re-break it out as bulk_nullable and label aren't required here.